src/__tests__/Button.spec.tsx

```diff
@@ -0,0 +1,12 @@
+import { shallow } from 'enzyme';
+import 'jest-enzyme';
+import * as React from 'react';
+
+import { Button } from '../Button';
+
+describe('Button component', () => {
+  it.only('renders', () => {
```

Comment:
I expect to find problems as we try to use this Button in Studio and we will add tests those scenarios.